aequilibrae/paths/network_skimming.py

```diff
@@ -68,16 +68,16 @@ def execute(self):
             self.skimming.emit(["text skimming", "Saving Outputs"])
             self.skimming.emit(["finished_threaded_procedure", None])
 
-    def func_skim_thread(self, O, all_threads):
```

Comment:
this seems like the same function as func_assign_thread, with exactly the same mistake (0 -> orig) in both.

Could be extracted and then future mistakes fixed in only one place :)

Reply:
The underlying function is actually quite different.